.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+me: Bug Report
+about: If things aren't working as expected.
+
+---
+
+## Bug Report
+
+<!-- 
+Thanks for filing an issue! Before hitting the button, please answer these questions.
+
+Fill in as much of the template below as you can. If you leave out information, we can't help you as well.
+-->
+
+**What did you do?**
+A clear and concise description of the steps you took (or insert a code snippet).
+
+**What did you expect to see?**
+A clear and concise description of what you expected to happen (or insert a code snippet).
+
+**What did you see instead? Under which circumstances?**
+A clear and concise description of what you expected to happen (or insert a code snippet).
+
+
+**Environment**
+* operator-sdk version:
+
+  insert release or Git SHA here
+
+* Kubernetes version information:
+
+  insert output of `kubectl version` here
+
+* Kubernetes cluster kind: 
+
```

Comment:
I think we should add a field about using ansible/go/helm. Don't have a good idea on wording though. Maybe `Are you writing your operator in ansible, go, or helm?`.